packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 9.0.7
+
+* [swift] Changes all ints to int64.
```

Comment:
Let's note here explicitly that existing code may need to be updated, like we've done for code-breaking changes to experimental generators in the past (see 7.2.0 for example).

Reply:
forgot to push that up earlier.